packages/services/hmi-server/src/main/resources/application.properties

```diff
@@ -29,5 +29,5 @@ xdd-extraction-service/mp-rest/url=https://xdddev.chtc.io/askem
 ########################################################################################################################
 # Native container configurations
 ########################################################################################################################
-quarkus.native.container-build=true
+quarkus.native.container-build=false
 quarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:22.3.0-java17
```

Comment:
Why this needs to be set here, but we do not use it in the `Dockerfile`.
Isn't it confusing to have this builder reference in two places?

Reply:
This can be removed completely. Left it in for now just in case. @dgauldie thoughts

Reply:
@dgauldie ????

Reply:
That is for when you run `quarkus` at the command-line at dev time.  When set to true it allows running Quarkus code without needing Java - useful for building native containers and the like, but can be false to have the host use the resident Java/GraalVM - at least that is my brief understanding from `https://quarkus.io/guides/building-native-image`

Reply:
I believe this is only relevant when building images, but will need @dgauldie to confirrm.

Reply:
we can remove entirely